connect/connect-cloudevents/message-adapter-cloudevents/src/main/resources/schemas/PublishMessageRequest.schema.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
💭 - Comparing the `message-adapter-cloud` events with the oder modules in `connect-cloud events` I saw that neither of them included any` /schemas` folder with json schemas inside. 

I would stick to the "pattern" and leave the generate files out of the commit as well. Though we should add them to the documentation once they are "live".

Reply:
These are not generated files. Actually these are the source files to generate Java POJOs from them. The other components in in this module serve a different purpose: mesage-adapter-cloudevents connects the internal java api from message domain to the cloudevents infrastrucutre whereas the other modules are the infrastructure. We can discuss if this should be moved to the message domain then. Already thought about this.

Reply:
Ah I see. Thanks for providing this context. :)
Then it is all okay for me 👍